libsolidity/codegen/YulUtilFunctions.cpp

```diff
@@ -1005,24 +1018,75 @@ string YulUtilFunctions::clearStorageRangeFunction(Type const& _type)
 {
 	string functionName = "clear_storage_range_" + _type.identifier();
 
-	solAssert(_type.storageBytes() >= 32, "Expected smaller value for storage bytes");
-
 	return m_functionCollector.createFunction(functionName, [&]() {
 		return Whiskers(R"(
-			function <functionName>(start, end) {
+			function <functionName>(start, end, firstInactiveInStart) {
```

Comment:
This is getting quite a weird function...
But this PR is blowing up a bit - what do you think about making "Implement resizing of packed storage arrays." its own PR :-)? Just changing the resize and clear function should work on its own and be testable, shouldn't it?